flux-tests/tests/neg/surface/const03.rs

```diff
@@ -1,18 +0,0 @@
-// Test definition and checking of const in struct
```

Comment:
What happened with this test? It is no longer working?

Reply:
whoops this was just some glitch on my part, let me restore!